src/SprykerSdk/Zed/AppSdk/Business/Manifest/Builder/ManifestBuilder.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Zed\AppSdk\Business\Manifest\Builder;
+
+use Generated\Shared\Transfer\ManifestRequestTransfer;
+use Generated\Shared\Transfer\ManifestResponseTransfer;
+use Generated\Shared\Transfer\MessageTransfer;
+
+class ManifestBuilder implements ManifestBuilderInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\ManifestRequestTransfer $manifestRequestTransfer
+     *
+     * @return \Generated\Shared\Transfer\ManifestResponseTransfer
+     */
+    public function createManifest(ManifestRequestTransfer $manifestRequestTransfer): ManifestResponseTransfer
+    {
+        $manifestResponseTransfer = new ManifestResponseTransfer();
+
+        $targetFilePath = $manifestRequestTransfer->getManifestPathOrFail();
+        $locale = $manifestRequestTransfer->getManifestOrFail()->getLocaleNameOrFail();
+
+        if ($this->isLocaleIsValid($locale) === false) {
+            $messageTransfer = new MessageTransfer();
+            $messageTransfer->setMessage('You have to enter a valid locale, example: en_US');
+
+            $manifestResponseTransfer->addError($messageTransfer);
+
+            return $manifestResponseTransfer;
+        }
+
+        $targetFile = $targetFilePath . $locale . '.json';
+
+        $manifest = $this->addDefaults();
+
+        $manifest['name'] = $manifestRequestTransfer->getManifestOrFail()->getNameOrFail();
+        $manifest['provider'] = $manifestRequestTransfer->getManifestOrFail()->getNameOrFail();
+
+        $this->writeToFile($targetFile, $manifest);
+
+        return $manifestResponseTransfer;
+    }
+
+    /**
+     * @return array<string, mixed>
+     */
+    protected function addDefaults(): array
+    {
+        $manifest = [
+            'description' => '',
+            'descriptionShort' => '',
+            'configureUrl' => '',
+            'categories' => [],
+            'assets' => [],
+            'resources' => [],
+            'pages' => [],
+            'label' => [],
+          ];
+
+        return $manifest;
```

Comment:
```suggestion
    protected function getManifest(ManifestRequestTransfer $manifestRequestTransfer): array
    {
        $manifestTransfer = $manifestRequestTransfer->getManifestOrFail();
        
        $manifest = [
            'name' => $manifestTransfer->getNameOrFail(),
            'provider' => $manifestTransfer->getNameOrFail(),
            'description' => '',
            'descriptionShort' => '',
            'configureUrl' => '',
            'categories' => [],
            'assets' => [],
            'resources' => [],
            'pages' => [],
            'label' => [],
          ];

        return $manifest;
```

Reply:
now got this and have implement the same.